src/NzbDrone.Core/Download/Clients/Porla/Porla.cs

```diff
@@ -0,0 +1,281 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using FluentValidation.Results;
+using NLog;
+using NzbDrone.Common.Disk;
+using NzbDrone.Common.Extensions;
+using NzbDrone.Common.Http;
+using NzbDrone.Core.Blocklisting;
+using NzbDrone.Core.Configuration;
+using NzbDrone.Core.Download.Clients.Porla.Models;
+using NzbDrone.Core.Localization;
+using NzbDrone.Core.MediaFiles.TorrentInfo;
+using NzbDrone.Core.Parser.Model;
+using NzbDrone.Core.RemotePathMappings;
+using NzbDrone.Core.Validation;
+
+namespace NzbDrone.Core.Download.Clients.Porla
+{
+    public class Porla : TorrentClientBase<PorlaSettings>
+    {
+        private readonly IPorlaProxy _proxy;
+
+        public Porla(IPorlaProxy proxy,
+                     ITorrentFileInfoReader torrentFileInfoReader,
+                     IHttpClient httpClient,
+                     IConfigService configService,
+                     IDiskProvider diskProvider,
+                     IRemotePathMappingService remotePathMappingService,
+                     ILocalizationService localizationService,
+                     IBlocklistService blocklistService,
+                     Logger logger)
+            : base(torrentFileInfoReader, httpClient, configService, diskProvider, remotePathMappingService, localizationService, blocklistService, logger)
+        {
+            _proxy = proxy;
+        }
+
+        public override string Name => "Porla";
+
+        public override IEnumerable<DownloadClientItem> GetItems()
+        {
+            var plist = _proxy.ListTorrents(Settings);
+
+            var items = new List<DownloadClientItem>();
+
+            // should probs paginate instead of cheating
+            foreach (var torrent in plist)
+            {
+                // we don't need to check the category, the filter did that for us, but we are checking anyway becuase why not :)
```

Comment:
If we don't need to because it's already filtered, then we don't need to filter again

Reply:
Yeah, at that point we are testing if porla works.
I'll remove it